sophia/src/parser/nq.rs

```diff
@@ -263,7 +264,7 @@ mod test {
         let mut d = HashSetDataset::new();
         let reader = io::Cursor::new(GENERALIZED_DOC);
         let res = STRICT.parse_read(reader).in_dataset(&mut d);
-        if let Err(Error(ParserError(_, location), _)) = res {
+        if let Err(StreamError::Source(Error(ParserError(_, location), _))) = res {
```

Comment:
Could be simplied as 
```suggestion
        if let Err(SourceError(Error(ParserError(_, location), _))) = res {
```
if you accept my suggestions on `StreamError`